setup.py

```diff
@@ -33,18 +33,18 @@
 DEV = [
     "flake8",
     "pytest",
-    "pandas",
+    "pandas==2.0.3",
```

Comment:
can you open an issue on this? so we don't forget we pinned it

Reply:
[opened issue](https://github.com/ploomber/jupysql/issues/890)